srcpkgs/vagrant/template

```diff
@@ -11,6 +11,7 @@ license="MIT"
 homepage="https://www.vagrantup.com/"
 distfiles="https://github.com/mitchellh/${pkgname}/archive/v${version}.tar.gz"
 checksum=4f8ce6ede5d47a2be1ce46a29d22d41d1f4ba9d99fe9496b2424f98ae4dab2d6
+patch_args="-Np1"
 nocross=yes
 
 archs="i686 x86_64"
```

Comment:
move it to  directly above hostmakedepends

Reply:
done

Reply:
@maxice8 PTAL